docs/adr/0032-l10n-i18n-improvements.md

```diff
@@ -0,0 +1,138 @@
+
+## Decision Drivers
+
+- Consistency across packages
+- Reliability
+- More robust and complete testing
+- Allowing `fxa-react` to manage its own l10n
+- More straightforward l10n in emails
+
+## Decision outcome 1: Use consistent FTL paths, names, and setup across packages
+
+`en-US` was FxA's default locale until early 2021 when the l10n team recommended we switch to `en`. `fxa-settings` still contains `en-US.ftl` files that will be updated to `en.ftl`.
+
+Historically, we did not deploy our `en` or `en-US` Fluent bundles, but we have since May 2022 to support any missing fallback text and for consistency across locales and environments.
+
+### `fxa-payments-server` setup changes
+
+`fxa-payments-server` commits a single file to [`public/locales/en-US/main.ftl`](https://github.com/mozilla/fxa/blob/main/packages/fxa-payments-server/public/locales/en-US/main.ftl) which is copied to the l10n repo. This differs from `fxa-settings` and `fxa-auth-server`, wherein we separate FTL files per component and then concatenate the files together to copy into the l10n repo.
+
+For consistency, better maintainability, and because we need to set up concatenation anyway (see the `fxa-react` decision), we'll split `fxa-payments-server`'s FTL file into per-component `en.ftl` files. This will require setting up something similar to `fxa-auth-server`'s `merge-ftl:test` script for test, and we can make sure those reference the same relative path. We can also rename `main.ftl` to become `payments.ftl`.
+
+#### Negative consequences
+
+Keeping strings in a single file that doesn't need an additional build step for testing or pushing to the l10n repo means the final output could feel more clear since a single committed file would reflect exactly what's going out.
+
+### Fluent configuration files
+
+Fluent configuration files are useful when the same repository needs to support different locales for different files or when there are complex paths involved, but because FxA manages its translations in a separate repository, it won't benefit. We'll remove `fxa-payments-server`'s config file.
+
+## Decision outcome 2: continue using fallback text and set up better testing strategies
+
+In theory, Fluent should always insert translated strings found in its bundles for us, including English strings. Because of this, we considered dropping our fallback text in favor of relying on Fluent.
+
+### Pros of dropping fallback text
+
+- Better maintainability and guarantee of consistency due to not needing duplicate strings (one for fallback text and one in an FTL file), in the codebase
+- It might feel more i18n-friendly to depend on Fluent instead of prioritizing English speaking users by explicitly setting English fallback text
+
+### Cons of dropping fallback text
```

Comment:
In theory it is possible that the browser won't be able to load the fluent bundle files after loading the page successfully, at which point the fallback text would be displayed.

Reply:
👍 Added another sentence to one of the cons.